Login page modifications

Koraytkn opened the pull request:
The login page is initialized using the structure that was determined by the Front-End team, there are some mock navigations that will be updated after the merging of different initial versions of the web application pages.

gokayyildiz commented on learnify/web/src/App.js:
```diff
@@ -1,7 +1,7 @@
 import './App.css';
 import Header from './pages/Header';
-import SignInForm from './pages/SignInForm';
+import LogInForm from './pages/LogInForm';
```
maybe we can use as Login instead of LogIn

gokayyildiz reviewed (changes requested):
The paddings and placing of the texts should comply with the Sign-Up page 
![signup](https://user-images.githubusercontent.com/42673110/197874174-7a0d52e2-76af-43ff-95ac-a7de8ddcd6f4.png)
![signin](https://user-images.githubusercontent.com/42673110/197874190-89782717-0c0b-42c1-bd52-4aa4770ac67f.png)

the titles for email and password should be on the left side like the one in Sign Up, there should not be that much spacing between boxes and the buttons at the top.

xltvy commented:
Also we can redesign the forgot password button and make it look like the Log In button: same shape but for example in white color, and place it below Log In button.

xltvy reviewed (changes requested):
Also we can redesign the forgot password button and make it look like the Log In button: same shape but for example in white color, and place it below Log In button.

Koraytkn commented:
I have updated the Login page considering the review items both by @gokayyildiz and @xltvy, accordingly I have updated the forgot password navigation button; changed the layout component of Login page and updated labels in the form.

xltvy reviewed (changes requested):
In App.js line 5 you added LoginForm import line:
`import LoginForm from './pages/LoginForm';`
However under the pages directory, the file is named `LogInForm.js`. Thus the code cannot be compiled.

gokayyildiz commented on learnify/web/src/App.js:
```diff
@@ -2,8 +2,8 @@ import { BrowserRouter, Routes, Route} from 'react-router-dom';
 import './App.css';
 import EmailVerificationPage from './pages/EmailVerificationPage';
 import Header from './pages/Header';
+import LoginForm from './pages/LoginForm';
```
import LoginForm from './pages/LoginForm' does this working since it gives me error because the file name is LogInForm.js in the repository

Koraytkn commented:
Now it should be fine, when I first changed the filename in my editor git support of VS code did not realize the name change as a modification.
